### PR TITLE
Upgrade google signin

### PR DESCRIPTION
**Pull request by misosviso**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,14 +4,17 @@ import './index.css';
...
+import { GoogleOAuthProvider } from '@react-oauth/google';
 import {ApolloProvider} from 'react-apollo';
 
 const render = () => {
   const App = require('./App').default;
   ReactDOM.render((
-    <ApolloProvider store={store} client={apolloClient}>
-      <App />
-    </ApolloProvider>
+    <GoogleOAuthProvider  clientId='914978031481-bk8e8bj1ur0vhq4qlh7n7875drin9r0e.apps.googleusercontent.com'>
```
Comment (sodik82) [inline src/index.js]: is it process.env.REACT_APP_CLIENT_ID ?

File: angular/src/app/login-page/login-page.component.ts [hunk 2]
```diff
@@ -38,11 +37,25 @@ export class LoginPageComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  initGoogle() {
+    window["google"].accounts.id.initialize({
```
Comment (sodik82) [inline angular/src/app/login-page/login-page.component.ts:41]: is there an easy way to add hosted_domain ? (it was not possible with the last lib)

File: src/index.js (hunk 1 above)
Comment (misosviso) [reply src/index.js]: Yes, it is indeed

File: angular/src/app/login-page/login-page.component.ts (hunk 2 above)
Comment (misosviso) [reply angular/src/app/login-page/login-page.component.ts:41]: according to the documention no

**Pull request merged**